dags/openshift_nightlies/config/install/aws/sdn-small.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Can we just get rid of OpenShiftSDN dags?

Reply:
done